src/MakePotCommand.php

```diff
@@ -295,7 +298,7 @@ protected function get_main_file_data() {
 	 *
 	 * @return bool True on success, false otherwise.
```

Comment:
That's now incorrect with this change.

Instead of making `translations` a property of the `MakePotCommand`, I think we can just use `$translations = new Translations()` and return that object at the end (like you did already).

We could get rid of `set_default_headers()` and move these ~10 lines into `makepot()`.

Also, since `makepot()` doesn't really create a POT file anymore, we could call it `extract_strings()` or something.